policyengine_us/parameters/gov/states/mi/tax/income/credits/eitc/rate/additional.yaml

```diff
@@ -0,0 +1,13 @@
+description: Michigan provides this maximum amount of tax credits under the state EITC.
+metadata:
+  unit: currency-USD
+  label: Michigan EITC max amount
+  reference:
+    - title: Government Michigan - Earned Income Tax Credit for Working Families
```

Comment:
this reference does not mention the 24%

Reply:
@MaxGhenis as per [law](https://casetext.com/statute/michigan-compiled-laws/chapter-206-income-tax-act-of-1967/income-tax-act-of-1967/section-206272-effective-91-days-after-adjournment-of-the-2023-regular-session-sine-die-earned-income-tax-credit-amount-equal-to-federal-credit-refund), the filer is entitled for a 24% additional credit amount in 2022 but it is up to the department to provide this extra amount and it is not mentioned on the tax form: 
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/c0b1e888-13e1-463b-8639-b52032c401e0)

Reply:
Makes sense that this would be absent from the form. Let's include it per the law.